### DISCUSSION TITLE
Custom function with noescape filter

### DISCUSSION THREAD

**Discussion by dragomano** [nette/latte#335] [Q&A]
Hello. I'm trying to implement a custom function using noescape filter but this filter does not work:

```php
$latte->addFunction('fa-icon', function (string $name) use ($latte): string {
        $icon = '<i class="fas fa-' . $name . '"></i>';
	return $latte->invokeFilter('noescape', [$icon]);
});
```
Is there any solution? I don't want to add `|noescape` for each `fa-icon` function in templates.

**Comment by dg** [comment]
Use `Latte\Runtime\Html` object as described here https://latte.nette.org/en/develop#toc-disabling-auto-escaping-of-variable
